Content.Shared/_RMC14/Pulling/RMCPullingSystem.cs

```diff
@@ -386,7 +390,9 @@ public override void Update(float frameTime)
                 continue;
             }
 
-            if (_mobState.IsDead(pulling))
+            if (_mobState.IsDead(pulling)
+                && !(TryComp<XenoComponent>(pulling, out var xeno)
+                && xeno.Role.Id == "CMXenoLarva"))
```

Comment:
probably better to have a ``IgnoreBlockPullingDeadComponent`` and give it to larva instead of hardcoding it to larva

Reply:
Done